### DISCUSSION TITLE
need help for install

### DISCUSSION THREAD

**Discussion by Ponponpostprod** [TheAcharya/MarkersExtractor#84] [Q&A] [answered]
I, i'm really interested as editor on FCPX, to try your app. But I don't understand how to install it. I need help in a more detailed way. I've download the markers-extractor-cli-0.3.2.zip. If i extract this file i have the markers-extractor.cli but what have i to do with this ?

**Comment by orchetect** [comment]
It's a command line interface utility (CLI). It's meant to run from the Terminal on macOS. We don't have any step-by-step guides because the assumption is that users will have familiarity with the use of the Terminal. But detailed guides could be added in future.

We are also developing an open-source GUI application called Marker Data that uses MarkersExtractor internally. It's not public yet but that would probably work better in your case. I think we're targeting February or March for a public release.

**Comment by IAmVigneswaran** [comment, marked as answer]
@Ponponpostprod Thank you for taking interest in trying out MarkersExtractor! 

While our Marker Data is still a few months away from public release, you can still utilise the CLI binary for your workflow. 

If you have download the zip package, you can follow this basic [example](https://github.com/TheAcharya/MarkersExtractor?tab=readme-ov-file#examples).

Let me know if that helps.

**Comment by IAmVigneswaran** [comment]
@Ponponpostprod Marker is now live. You can check it out.

https://github.com/TheAcharya/MarkerData
